packages/arb-token-bridge-ui/src/components/common/atoms/Toast.tsx

```diff
@@ -1,9 +1,27 @@
 import { ToastContainer, toast } from 'react-toastify'
 
-export const errorToast = (message: string) => {
+import { ExternalLink } from '../ExternalLink'
+import { GET_HELP_LINK } from '../../../constants'
+
+export const networkConnectionWarningToast = () =>
```

Comment:
should we have this in `Toast` file? maybe it's better if it's just right where it's needed

Reply:
But TransferPanel is such a lengthy file
Moving it there would bury this function and reduce discoverability
If you use "Go to Definition", it'd scroll all the way up away from where it is used
IMO it's a worse experience

Reply:
I have to disagree here. Let's put it somewhere where you'd put it if Toast was imported from the component library as a package.